components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -249,6 +249,7 @@ func (w *NotebookWebhook) Handle(ctx context.Context, req admission.Request) adm
 
 	}
 
+	print("controllers/notebook_webhook")
```

Comment:
Hey Jiri, this print statement looks more like the Python way of printing rather than Go, which requires the `fmt` package. Also, keep in mind that this print will be part of the controller logs.

Reply:
This works too, Go has a built-in `print` function, here's the reference for it https://go.dev/ref/spec#Bootstrapping; you're not supposed to use it, but here I just want to make a change to trigger coverage reporting.